<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CriarUsuario</class>
 <widget class="QDialog" name="CriarUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome de usuário:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="NomeUsuario"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Senha principal:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="Senha1">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Confirma senha:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="Senha2">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="BotaoVoltar">
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="BotaoCriarUsuario">
     <property name="text">
      <string>Criar usuário</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotaoVoltar</sender>
   <signal>clicked()</signal>
   <receiver>CriarUsuario</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
